### PR TITLE
master-qa-54808

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-54808

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro [hunk 1]
```diff
@@ -262,9 +262,39 @@ definition {
 		var key_currentAssignee = "${currentAssignee}";
 		var key_currentDateDay = DateUtil.getCurrentDate();
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro]: Hi Melody,
Thank you for making these changes. I noticed what was going on, and it seems that we have a method to format dates on DateUtil. So, I changed only the value of key_currentDateDay and it was all good (+ adding key_ to the other variables).

var key_currentDateDay = DateUtil.getFormattedDate("0", "dd");

**Comment by barrosdennys** [comment]
I sent a fix to BChan:
https://github.com/brianchandotcom/liferay-portal/pull/82909/files

Thank you!!

**Pull request closed without merging**